releng/org.eclipse.xtext.target/org.eclipse.xtext.target-latest.target

```diff
@@ -86,11 +87,6 @@
 		<repository location="http://download.eclipse.org/modeling/emf/emf/builds/nightly/latest"/>
 	</location>
 
-	<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="false" type="InstallableUnit">
-		<unit id="org.eclipse.draw2d.feature.group" version="0.0.0"/>
```

Comment:
? what has changed here?

Reply:
Moved to be consumed from SimRel. We don't rely on a specific version. SimRel contains org.eclipse.draw2d.feature.group=3.10.100.201606061308